source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,190 @@
+
+Do note that the current setup is only reachable on this machine.
+
+Testing the server locally
 --------------------------
 
-Using Unbound as the resolver for a home network.
+To verify that the server works correctly it’s a good idea to test it before committing the entire network to it. Luckily we can test this on the machine that you installed Unbound on (locally) and from any other machine (remotely) that will be using the resolver after we expose Unbound to the network.
+
+The command for local testing is:
+
+.. code-block:: bash
+
+	dig example.com @127.0.0.1
+
+Here we tell the :command:`dig` tool to look up the IP address for example.com, and to ask this information to the server running at the IP address ``127.0.0.1``, which is where our Unbound machine is running by default.
+We can verify that Unbound has indeed answered our query instead of the default resolver that is present on Ubuntu by default. In the output of every :command:`dig` command there is ``ANSWER SECTION`` which specifies the server which has answer the query under ``SERVER`` entry. The entry should be ``;; SERVER: 127.0.0.1#53(127.0.0.1)``.
+
+In the next section we will be disabling the default Ubuntu resolver. To verify that we do it correctly there it is useful to know the address of the default resolver as a baseline. For this baseline, we also use a :command:`dig` query but time without specifying an IP address which the uses the machines default DNS resolver.
+
+.. code-block:: bash
+
+	dig example.com
+
+While the response should be the same, the ``SERVER`` in the response should look like ``;; SERVER: 127.0.0.53#53(127.0.0.53)``. Note that the final IPv4 digit is 53 and not 1, as with our Unbound instance.
+
+Setting up for a single machine
+-------------------------------
+
+Now that we have configured and tested our Unbound server, we can tell our machine to use it by default. The nameserver (i.e. resolver) your machine uses by default is defined in :file:`/etc/resolv.conf`.
+While just changing this file will work as long as the machine doesn't reboot, the more permanent and better solution is to replace the file with our own. The reason for this is that the :file:`resolv.conf` file is a `symbolic link`. We will remove the link and create a new file ourselves.
+
+.. code-block:: bash
+
+	rm /etc/resolv.conf
+
+With your favourite text editor (if you don't have a favourite you could use :command:`nano`), we can then create a new file with the same name and fill it with the IP address that our Unbound instance is running at, and we include the :option:`edns0` option as this enables header extensions used in DNSSEC and is an overall standard used in DNS nowadays. |br|
+So with :file:`nano /etc/resolv.conf` we create the new file and enter:
+
+.. code-block:: bash
+
+	nameserver 127.0.0.1
+	options edns0
+
+We then need to stop and disable the currently running pre-installed resolver. Note that you lose connectivity to the web until the next step.
```

Comment:
You don't lose connectivity to the web. You can't resolve any domain names until the next step. Or, you lose DNS, or however you wanna phrase it.